v2/src/components/recipeBoxes/guides.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
The alignment for the header and the column content is off

<img width="506" alt="Screenshot 2023-03-13 at 6 38 00 PM" src="https://user-images.githubusercontent.com/18233774/224710877-e9e24202-7ffe-4085-92f0-d00369d3e198.png">